controllers/insights/index.js

```diff
@@ -11,15 +11,29 @@ const {
 } = require('../../middleware/inject-content');
 const { buildArchiveUrl } = require('../../modules/archived');
 const { localify } = require('../../modules/urls');
+const { buildPagination } = require('../../modules/pagination');
 
 const router = express.Router();
 
-router.get('/', injectHeroImage('insights-letterbox-new'), injectCopy('insights'), injectResearch, (req, res) => {
+router.get('/', injectHeroImage('insights-letterbox-new'), injectCopy('insights'), injectResearch(), (req, res) => {
     res.render(path.resolve(__dirname, './views/insights-landing'), {
         researchArchiveUrl: buildArchiveUrl(localify(req.i18n.getLocale())('/research'))
     });
 });
 
+router.get(
+    '/documents',
+    injectHeroImage('insights-letterbox-new'),
+    injectCopy('insights.documents'),
+    injectResearch('documents', 10),
```

Comment:
Not sure if this is the best way to do this – could just make another middleware and control this there, perhaps?

Reply:
I'm starting to find the `inject*` pattern a little awkward in cases where we only use it in a single controller. One two many levels of abstraction. It might be easier to inline the content-api calls into this method. Otherwise yeah a separate middleware would work.

Reply:
Done!